src/arch/mod.rs

```diff
@@ -349,7 +335,7 @@ mod tests {
     #[test]
     fn char_struct() {
         let char_struct = type_for_size(5);
-        assert_eq!(char_struct.alignof().unwrap(), 4);
+        assert_eq!(char_struct.alignof().unwrap(), 1);
         assert_offset(vec![Type::Int(true), Type::Char(true)], 1, 4);
         assert_eq!(char_struct.sizeof().unwrap(), 5);
     }
```

Comment:
Could you add one last test that makes sure this has the correct alignment? Something like

```
#[test]
fn align() 
   let ty = struct_for_types(vec![Int(True), Int]);
   assert_eq!(ty.alignof(), Ok(8));
}
```

Reply:
that should be an alignment of 4, correct?

Reply:
Whoops, I meant for the first type to be a pointer, not an int.